pkg/sm/sm.go

```diff
@@ -200,38 +199,31 @@ func (smb *ServiceManagerBuilder) Build() *ServiceManager {
 }
 
 func (smb *ServiceManagerBuilder) installHealth() error {
-	if len(smb.HealthIndicators) == 0 {
-		return nil
-	}
-
 	healthz := h.New()
 	logger := log.C(smb.ctx).Logger
 
 	healthz.Logger = l.New(logger)
 	healthz.StatusListener = &health.StatusListener{}
 
-	tresholds := make(map[string]int64)
+	thresholds := make(map[string]int64)
 
 	for _, indicator := range smb.HealthIndicators {
-		if configurableIndicator, ok := indicator.(health.ConfigurableIndicator); ok {
-			if settings, ok := smb.HealthSettings[configurableIndicator.Name()]; ok {
-				configurableIndicator.Configure(settings)
-			} else {
-				configurableIndicator.Configure(health.DefaultIndicatorSettings())
-			}
+		settings, ok := smb.cfg.Health.Indicators[indicator.Name()]
```

Comment:
you should probably also handle partcially configured settings that are present in the map - meaning "complete" them with default values

Reply:
Discussed and it seems that now partically configuted indicator will result in confiuration validation error - which is also ok